src/rpc/multipass.proto

```diff
@@ -156,7 +157,8 @@ message InstanceStatus {
         STOPPED = 3;
         DELETED = 4;
         DELAYED_SHUTDOWN = 5;
-        UNKNOWN = 6;
+        SUSPENDED = 6;
+        UNKNOWN = 7;
```

Comment:
Should we maybe avoid breaking compatibility like this?

Reply:
Yeah, we haven't been too worried about compatibility since the one and only client and daemon have been in lock step, but yes, at some point care should be taken when updating things like this.  It would probably be better to have `UNKNOWN` first in the enum, and then it will look cleaner when adding new states, etc.